src/common/product_models/heatmap_common_models.py

```diff
@@ -52,6 +53,29 @@ class ColumnType(str, Enum):
     """ A boolean. """
 
 
+# Maps a heatmap column type to a genome attributes column type
+HEATMAP_TO_ATTRIBS_MAPPING = {
```

Comment:
Does this need to be a public variable or should it be private?

Reply:
👍 make it private.

Reply:
👍 make it private.

Reply:
:+1: